Assets/Scripts/Models/Area/World.cs

```diff
@@ -51,6 +51,11 @@ public World(int width, int height, int depth)
         // Creates an empty world.
         SetupWorld(width, height, depth);
         int seed = UnityEngine.Random.Range(0, int.MaxValue);
```

Comment:
Yes please from someone who starts a lot from _World this is very needed :D.  Also it shouldn't be from 0 to MaxValue shouldn't it be `int seed = UnityEngine.Random.Range(int.MinValue, int.MaxValue)` ? shouldn't we include negatives